AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Invite/Send.cshtml

```diff
@@ -0,0 +1,56 @@
+﻿@model AllReady.Areas.Admin.ViewModels.Invite.InviteViewModel
+
+@{
+    string formAction;
+    if (Model != null && Model.InviteType == AllReady.Areas.Admin.ViewModels.Invite.InviteType.EventManagerInvite)
+    {
+        ViewData["Title"] = "Send Event Manager Invite";
+        formAction = "SendEventManagerInvite";
+    }
+    else
+    {
+        ViewData["Title"] = "Send Campaign Manager Invite";
+        formAction = "SendCampaignManagerInvite";
+    }
+}
+
```

Comment:
if possible, try to move these conditional checks to the controller, and then return the Title of the page and the FormAction as properties on the view model. That way, both the Title, and the FormAction are unit testable, and the view does not contain conditional logic. We want to keep our views as simple and "stupid" as possible.